docs/index.md

```diff
@@ -0,0 +1,20 @@
+# Documentation
+
+Welcome to the [scp-machine](https://github.com/ostis-ai/scp-machine) documentation! This comprehensive guide is designed to help you navigate the features, functionalities, and applications of scp-machine, a C++ software interpreter of programs in the basic programming language of the [**OSTIS Technology**](https://github.com/ostis-ai).
+
+## What is scp-machine?  
+
+scp-machine is a **software interpreter** designed to execute programs written in the basic programming language of the **OSTIS Technology**. It extends the capabilities of the [**sc-machine**](https://github.com/ostis-ai/sc-machine), which serves as a platform for developing intelligent agents based on knowledge representation and processing. scp-machine interprets programs that are structured as knowledge graphs, utilizing the *SCP (Semantic Code Programming) Language*.
+
+Table of contents:
+
+- [Quick Start](quick_start.md) - *get up and running with scp-machine quickly*
+- **Build Instructions** - *guidelines for building the project, configuring settings*
+    * [Quick Start for Developers](build/quick_start.md) - *get up and start developing scp-machine quickly*
+    * [Build System](build/build_system.md) - *how to build the project and use it as a library*
+    * [CMake Flags](build/cmake_flags.md) - *description of CMake flags used to configure scp-machine*
+- **Development** - *guidelines for contributing to development*
+    * [Contributing Guide](https://github.com/ostis-ai/scp-machine/blob/main/CONTRIBUTING.md) - *guide for those who wants to make contribution into scp-machine*
```

Comment:
this url leads to github page, not documentation

Reply:
Yes, there's no other way to fix it